### DISCUSSION TITLE
Luxembourgish Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#754] [Language Leaders]
@mojikosu Would you be interested in being a language leader for Luxembourgish?

**Comment by mojikosu** [comment]
@synesthesiam Wow, thanks for asking. Yes, I would be delighted.

**Reply by synesthesiam** [reply]
Awesome! Adding you now :slightly_smiling_face:

**Comment by kitty99hub** [comment]
Hi.
@synesthesiam
@mojikosu
I speak and write Luxembourgish and would like to contribute to the development of the language in Home Assistant.
Unfortunately, I have absolutely no idea how this works on GitHub and I'm having a hard time understanding it at the moment :)
If help is still needed for the Luxembourgish language and someone would be willing to introduce me to the workflow, I would be very happy to become part of the community.
Many thanks

**Reply by mojikosu** [reply]
@kitty99hub Moien! :) 
Could this be of help to you? These are the intructions to get started with language contributions: https://github.com/home-assistant/intents/blob/main/docs/codespace/README.md
In essence: 
1) you create a cloudspace coding environment (it lives as a cloud environment on github). 
2) You apply your code changes in that could environment and then you submit it for approval. 
3) If your changes were approved and integrated into the main source code, you may safely delete your cloudspace branch  you created your modifications in (they always get a fancy name assigned by github), because they have been merged into the main code.
4) For every new contribution effort you create a new cloudspace coding environment from the main branch and repeat the steps.

**Comment by synesthesiam** [comment]
@mojikosu when you get a chance, could you please look into translating the `HassGetState` sentences/responses?

**Reply by mojikosu** [reply]
Hi, I am on it. Latest pull request does not go through because another language (gu) fails.

**Reply by synesthesiam** [reply]
Sorry about that, should be fixed now :+1:

**Reply by mojikosu** [reply]
No worries! Thanks. I won't be able to finish this tonight, anyway. My goal is to have all getstate files ready by Thursday night.
